src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
```

Comment:
Link to the stylesheet should have an extension of `.css` instead of `.scss`. SCSS files are preprocessed and should be compiled to CSS before being linked in HTML.